src/mapcss/__init__.py

```diff
@@ -398,16 +413,22 @@ def parse(self, css=None, clamp=True, stretch=1000, filename=None, static_tags={
                             else:
                                 stylez['z-index'] = res
         except TypeError:
+            # TODO: Better error handling here
             pass
 
+        # Group MapCSS styles by object type: 'area', 'line', 'way', 'node'
```

Comment:
it'd better to drop this superfluous syntax altogether, please see a discussion in https://github.com/organicmaps/organicmaps/issues/4137#issuecomment-1403638671

but there are some tricky cases that rely on the current unobvious inheritance behavior of area/line/etc - needs to be fixed in mapcss files